src/routes/organizations/entities/create-organizations.dto.entity.ts

```diff
@@ -0,0 +1,11 @@
+import { Organization } from '@/datasources/organizations/entities/organizations.entity.db';
+import { ApiProperty } from '@nestjs/swagger';
+import { number } from 'zod';
+
+export class CreateOrganizationResponse {
+  @ApiProperty({ type: String })
+  public readonly name!: Organization['name'];
+
+  @ApiProperty({ type: number })
```

Comment:
`z.number` is a validation method. (I'm fairly certain we don't need specify the `type` of primitives though.)

```suggestion
  @ApiProperty({ type: Number })
```

Reply:
The zod method was mistakenly imported. When fetching a variable type from another type, we need to explicitly specify the type.